index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sign-Up Form</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/06aa7828ae.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <main>
+        <header>
+            <div class="account">
+                <div class="acc">Already have an Inuit account? <br> <a class="acc-link" href="#">I forgot my User ID or
+                        Password</a></div>
+
+                <div class="ol"><i class="fas fa-lock"></i> Sign In</div>
+            </div>
+        </header>
+        <section>
+            <div class="logo"><img src="image/logo.png" alt="logo"></div>
```

Comment:
- [ ] It shows the Mint logo and text (logo on the left, text on the right) [link to page](https://accounts.intuit.com/signup.html)

- [ ] It shows the Quickbooks logo and text (logo on the left, text on the right) [link to page](https://accounts.intuit.com/signup.html)

- [ ] It shows the Turbotax logo and text (logo on the left, text on the right) [link to page](https://accounts.intuit.com/signup.html)